test/test_utils/ec2.py

```diff
@@ -911,7 +911,42 @@ def post_process_mxnet_ec2_performance(connection, log_location):
         raise ValueError("total: {}; n: {} -- something went wrong".format(total, n))
 
 
-@retry(reraise=True, wait=wait_fixed(60), stop=stop_after_attempt(5))
+def kill_background_processes_and_run_apt_get_update(ec2_conn):
+    """
+    The apt-daily services on the DLAMI cause a conflict upon running any "apt install" commands within the first few
+    minutes of starting an EC2 instance. These services are not necessary for the purpose of the DLC tests, and can
+    therefore be killed. This function kills the services, and then forces "apt-get update" to run in the foreground.
+
+    :param ec2_conn: Fabric SSH connection
+    :return:
+    """
+    apt_daily_services_list = ["apt-daily.service", "apt-daily-upgrade.service", "unattended-upgrades.service"]
+    apt_daily_services = " ".join(apt_daily_services_list)
+    ec2_conn.run(f"sudo systemctl stop {apt_daily_services}")
+    ec2_conn.run(f"sudo systemctl kill --kill-who=all {apt_daily_services}")
+    num_stopped_services = 0
+    # The `systemctl kill` command is expected to take about 1 second. The 60 second loop here exists to force
```

Comment:
nit: Did you want to improve this wait while we're here? Or should we consider this out of scope for this PR?

Reply:
I agree, but I'd rather just merge this in to unblock all the other failures we've been seeing.